### PR TITLE
How-to fixes

### PR DESCRIPTION
**Pull request by jkanem**
- [ ] I have added the tests to cover my changes. N/A; this is documentation
- [x] I have updated the documentation and CHANGELOG accordingly.
- [x] I have read the CONTRIBUTING document.

### CODE REVIEW

**Comment by jkanem** [comment]
Here are the how-to notebooks which need checking:
- - [ ] collection
- - - [ ] query_electron_execution_result
- - - [ ] query_lattice_execution_result
- - - [ ] query_multiple_lattice_execution_results
- - [ ] config
- - - [ ] customization
- - [ ] execution
- - - [ ] cancel_dispatch
- - - [ ] choosing_conda_environments
- - - [ ] choosing_executors
- - - [x] creating_custom_executors
- - - [ ] execute_individual_electron
- - - [ ] execute_lattice
- - - [ ] execute_multiple_lattices
- - - [ ] execute_sublattice
- - - [ ] synchronize_lattice
- - [ ] orchestration
- - - [ ] add_constraints_to_lattice
- - - [ ] add_electron_to_lattice
- - - [ ] construct_c_task
- - - [ ] construct_electron
- - - [ ] construct_lattice
- - - [ ] construct_lepton
- - - [ ] visualize_lattice
- - [ ] status
- - - [ ] query_electron_execution_status
- - - [ ] query_lattice_execution_status
- - - [ ] query_lattice_execution_time

**Comment by jkanem** [comment]
Fixes https://github.com/AgnostiqHQ/covalent-staging/issues/460

File: doc/source/how_to/execution/choosing_conda_environments.ipynb [hunk 1]
```diff
@@ -96,7 +87,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    ".. note:: Ensure that the covalent dispatcher server has been started properly before attempting to execute the lattice. Custom ports can be assigned via `covalent start --port [port no.]` which then needs to be modified accordingly in the `dispatcher_addr` parameter below. More details discussed in [How to add constraints to lattices](../orchestration/add_constraints_to_lattice.ipynb)."
+    ".. note:: Ensure that the covalent server has been started properly before attempting to execute the lattice."
```
Comment (scottwn) [inline doc/source/how_to/execution/choosing_conda_environments.ipynb:90]:
```suggestion
    ".. note:: Ensure that Covalent services have been started properly before attempting to execute the lattice."
```

File: doc/source/how_to/execution/choosing_executors.ipynb [hunk 2]
```diff
@@ -85,7 +81,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    ".. note:: Ensure that the covalent dispatcher server has been started properly before attempting to execute the lattice. Custom ports can be assigned via `covalent start --port [port no.]` which then needs to be modified accordingly in the `dispatcher_addr` parameter below. More details discussed in [How to add constraints to lattices](../orchestration/add_constraints_to_lattice.ipynb)."
+    ".. note:: Ensure that the covalent server has been started properly before attempting to execute the lattice."
```
Comment (scottwn) [inline doc/source/how_to/execution/choosing_executors.ipynb:84]:
```suggestion
    ".. note:: Ensure that Covalent services have been started properly before attempting to execute the lattice."
```

File: doc/source/how_to/execution/execute_individual_electron.ipynb [hunk 3]
```diff
@@ -32,14 +32,7 @@
...
    "source": [
-    ".. note:: It is important to ensure that the correct dispatcher server with the proper port value is assigned in the `lattice` constraint."
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    ".. tip:: We can set the results directory in the `lattice` decorator. Consult [How to add constraints to lattices](../orchestration/add_constraints_to_lattice.ipynb) for more details."
+    ".. note:: Ensure that the covalent server has been started properly before attempting to execute the lattice."
```
Comment (scottwn) [inline doc/source/how_to/execution/execute_individual_electron.ipynb:35]:
```suggestion
    ".. note:: Ensure that Covalent services have been started properly before attempting to execute the lattice."
```

File: doc/source/how_to/execution/execute_lattice.ipynb [hunk 4]
```diff
@@ -39,7 +35,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    ".. note:: Ensure that the covalent dispatcher server has been started properly before attempting to execute the lattice. Custom ports can be assigned via `covalent start --port [port no.]` which then needs to be modified accordingly in the `dispatcher` constraint. More details discussed in [How to add constraints to lattices](../orchestration/add_constraints_to_lattice.ipynb)."
+    ".. note:: Ensure that Covalent has been started before attempting to execute the lattice."
```
Comment (scottwn) [inline doc/source/how_to/execution/execute_lattice.ipynb:38]:
```suggestion
    ".. note:: Ensure that Covalent services have been started before attempting to execute the lattice."
```

**Review by scottwn — CHANGES_REQUESTED** [review]

File: doc/source/how_to/execution/choosing_conda_environments.ipynb (hunk 1 above)
Comment (jkanem) [reply doc/source/how_to/execution/choosing_conda_environments.ipynb]: Fixed

File: doc/source/how_to/execution/execute_lattice.ipynb (hunk 4 above)
Comment (jkanem) [reply doc/source/how_to/execution/execute_lattice.ipynb]: Fixed

**Pull request merged**